env-config.js

```diff
@@ -15,90 +17,100 @@ export const PREFIXES = `
-export const SKIP_MU_AUTH_INITIAL_SYNC = process.env.SKIP_MU_AUTH_INITIAL_SYNC == 'true' ? true : false ;
-export const VIRTUOSO_ENDPOINT = process.VIRTUOSO_ENDPOINT  || 'http://virtuoso:8890/sparql';
-export const MU_AUTH_ENDPOINT = process.MU_AUTH_ENDPOINT || 'http://database:8890/sparql';
-
-//DIFFERENT ENDPOINT FOR PUBLICATION GRAPH
-export const PUBLICATION_VIRTUOSO_ENDPOINT = process.env.PUBLICATION_VIRTUOSO_ENDPOINT || VIRTUOSO_ENDPOINT;
-export const PUBLICATION_MU_AUTH_ENDPOINT = process.env.PUBLICATION_MU_AUTH_ENDPOINT || MU_AUTH_ENDPOINT;
-
-//FILES PUBLISHER
-export const SERVE_DELTA_FILES = process.env.SERVE_DELTA_FILES || false;
-export const LOG_OUTGOING_DELTA = process.env.LOG_OUTGOING_DELTA || false;
-export const DELTA_INTERVAL = process.env.DELTA_INTERVAL_MS || 1000;
-export const PRETTY_PRINT_DIFF_JSON = process.env.PRETTY_PRINT_DIFF_JSON == 'true';
-export const ERROR_GRAPH =  process.env.ERROR_GRAPH || 'http://mu.semte.ch/graphs/system/errors';
-export const RELATIVE_FILE_PATH = process.env.RELATIVE_FILE_PATH || 'deltas';
-export const FILES_GRAPH = process.env.FILES_GRAPH || 'http://mu.semte.ch/graphs/public';
-
-export const CACHE_CHUNK_STATEMENT = parseInt(process.env.CACHE_CHUNK_STATEMENT || 100);
-export const CACHE_CHUNK_ARRAY = parseInt(process.env.CACHE_CHUNK_ARRAY || 10);
-
-//LOGIN
-export const KEY = process.env.KEY || '';
-export const ACCOUNT = process.env.ACCOUNT || 'http://services.lblod.info/diff-consumer/account';
-export const ACCOUNT_GRAPH = process.env.ACCOUNT_GRAPH || 'http://mu.semte.ch/graphs/diff-producer/login';
-
-/*
- * END EXPERIMENTAL FEATURES
- */
+        this.ERROR_URI_PREFIX = 'http://redpencil.data.gift/id/publication-maintenance/error/';
+        this.JOBS_GRAPH = configData.JOBS_GRAPH || 'http://mu.semte.ch/graphs/system/jobs';
+        this.JOB_TYPE = 'http://vocab.deri.ie/cogs#Job';
+        this.TASK_TYPE = 'http://redpencil.data.gift/vocabularies/tasks/Task';
+        this.STATUS_BUSY = 'http://redpencil.data.gift/id/concept/JobStatus/busy';
+        this.STATUS_SCHEDULED = 'http://redpencil.data.gift/id/concept/JobStatus/scheduled';
+        this.STATUS_SUCCESS = 'http://redpencil.data.gift/id/concept/JobStatus/success';
+        this.STATUS_FAILED = 'http://redpencil.data.gift/id/concept/JobStatus/failed';
+        this.STATUS_CANCELED = 'http://redpencil.data.gift/id/concept/JobStatus/canceled';
+        this.ERROR_TYPE = 'http://open-services.net/ns/core#Error';
+        this.DELTA_ERROR_TYPE = 'http://redpencil.data.gift/vocabularies/deltas/Error';
+        this.ERROR_CREATOR_URI = configData.ERROR_CREATOR_URI || 'http://lblod.data.gift/services/delta-producer-publication-graph-maintainer';
+
+        //task operation of interest
+        this.HEALING_PATCH_PUBLICATION_GRAPH_TASK_OPERATION = 'http://redpencil.data.gift/id/jobs/concept/TaskOperation/deltas/healing/patchPublicationGraph';
```

Comment:
SCREAMING_SNAKE_CASE for const values, camel cases for  instance value. I think it will help to determine more easily what comes from the services file to the env variables. Also const values can be exported out from outside the Config class definition.